package/MDAnalysis/__init__.py

```diff
@@ -178,10 +178,10 @@
 _SELECTION_WRITERS: Dict = {}
 _CONVERTERS: Dict = {}
 # Registry of TopologyAttributes
-_TOPOLOGY_ATTRS: Dict = {}   # {attrname: cls}
-_TOPOLOGY_TRANSPLANTS: Dict = {}   # {name: [attrname, method, transplant class]}
-_TOPOLOGY_ATTRNAMES: Dict = {}   # {lower case name w/o _ : name}
-
+_TOPOLOGY_ATTRS = {}  # {attrname: cls}
+_TOPOLOGY_TRANSPLANTS = {}   # {name: [attrname, method, transplant class]}
+_TOPOLOGY_ATTRNAMES = {}  # {lower case name w/o _ : name}
+_GUESSERS = {}
```

Comment:
```suggestion
_TOPOLOGY_ATTRS: Dict = {}  # {attrname: cls}
_TOPOLOGY_TRANSPLANTS: Dict = {}   # {name: [attrname, method, transplant class]}
_TOPOLOGY_ATTRNAMES: Dict = {}  # {lower case name w/o _ : name}
_GUESSERS: Dict = {}
```

We recently introduced type annotations to the library. Here we need to tell the type checker that these variables are dictionaries.